### PR TITLE
Changes to post a comment and view comment. Needs work to show certain comments

### PR DESCRIPTION
**Pull request by AndrewKishino**
Wanted to push the changes to post and view a comment. There will need to be a few changes but you should be able to make a comment and see the comment. As of now, all comments for all posts show up in every post. I made a lot of changes so feel free to tear apart what I have.

### CODE REVIEW

File: client/styles/post-styles.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.authorTags {
+  background-color: #EFEFEF;
+  padding: 8px;
+  display: inline-block;
+}
+
+.CodeMirror, .CodeMirror-scroll {
+    min-height: 200px !important;
+}
```
Comment (codydaig) [inline client/styles/post-styles.css]: New Line Please!

**Comment by Kevbyte** [comment]
Please merge my newupload pull request before merging this as I did make a few minor edits to post template

**Comment by codydaig** [comment]
LGTM

**Comment by lina** [comment]
lgtm

**Pull request merged**